### PR TITLE
Copy Edits to Readme

### PR DESCRIPTION
**Pull request by digitalbuddha**
Made the readme a little more robust. not sure if better or not :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
...
 [![Build status](https://travis-ci.org/apollographql/apollo-android.svg?branch=master)](https://travis-ci.org/apollographql/apollo-android)
 
-This is a Gradle plugin and set of libraries that generate Java code based on a GraphQL schema and query documents.
-The plugin uses `apollo-codegen` under the hood.
+Apollo-Android is a GraphQL compliant client that generates Java models from your standard GraphQL queries. These models give you a typesafe API to work with GraphQL servers. Apollo will help you keep your GraphQL query statements together, organized, and easy to access from Java.  Change a query and recompile your project - Apollo code gen will remake your data model.  Code generation also allows Apollo to read and unmarshal responses from the network without the need of any reflection (see code gen example below).   Feature versions of Apollo-Android will also work with AutoValue and other Value Object generators.
```
Comment (felipecsl) [inline README.md]: nit: `rebuild` instead of `remake`?

Comment (felipecsl) [inline README.md]: `Future* versions`

Comment (felipecsl) [inline README.md]: lowercase "value object"

**Comment by digitalbuddha** [comment]
changes applied

File: README.md [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
 [![Build status](https://travis-ci.org/apollographql/apollo-android.svg?branch=master)](https://travis-ci.org/apollographql/apollo-android)
 
-This is a Gradle plugin and set of libraries that generate Java code based on a GraphQL schema and query documents.
-The plugin uses `apollo-codegen` under the hood.
+Apollo-Android is a GraphQL compliant client that generates Java models from your standard GraphQL queries. These models give you a typesafe API to work with GraphQL servers. Apollo will help you keep your GraphQL query statements together, organized, and easy to access from Java.  Change a query and recompile your project - Apollo code gen will rebuild your data model.  Code generation also allows Apollo to read and unmarshal responses from the network without the need of any reflection (see code gen example below).   Future versions of Apollo-Android will also work with AutoValue and other value object generators.
```
Comment (marwanad) [inline README.md]: `   Feature` is there an extra space here?

File: README.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ public final class DroidDetails implements Query<Operation.Variables> {
 
 ## Consuming Code
 
-You can then use the genrated classes with Retrofit to make requests to your GraphQL API:
+You can then use the genrated classes with Retrofit to make requests to your GraphQL API, Apollo includes a `ApolloConverterFactory` which will tie into auto generated response mapping code.
```
Comment (marwanad) [inline README.md]: nit: new sentence instead of a comma?

File: README.md (hunk 2 above)
Comment (felipecsl) [inline README.md]: yep, double space

Comment (digitalbuddha) [inline README.md]: corrected to 2 from 3, good eye.

**Comment by digitalbuddha** [comment]
better or worse :-) ?

**Comment by felipecsl** [comment]
Looking better 👍

**Pull request merged**